### PR TITLE
bug: LT-1952 Update Third Party design for sub type & role

### PR DESCRIPTION
**Pull request by BenAndrew-Kainos**

### CODE REVIEW

File: applications/views/parties/base.py [hunk 1]
```diff
@@ -50,10 +50,10 @@ def init(self, request, **kwargs):
         self.object_pk = kwargs["pk"]
         application = get_application(request, self.object_pk)
         self.forms = self.form(application, self.strings, self.back_url)
-        if self.multiple_allowed:
-            self.data = None
-        else:
-            self.data = application[self.name]
+        if not self.multiple_allowed and "edit" in request.path:
```
Comment (efahy) [inline applications/views/parties/base.py]: This seems a bit haphazard to me - is there not a more definitive way to identify that it's an edit?  Happy to discuss

**Comment by efahy** [comment]
Please add a role column on the exporter application detail page

File: templates/applications/parties/third-parties.html [hunk 2]
```diff
@@ -24,12 +24,13 @@ <h1 class="govuk-heading-l">{% block title %}{% lcs 'third_parties.ThirdPartiesP
...
-					<th class="govuk-table__header" scope="col">{% lcs 'third_parties.ThirdPartiesPage.Variables.ADDRESS' %}</th>
-					<th class="govuk-table__header" scope="col">{% lcs 'third_parties.ThirdPartiesPage.Variables.COUNTRY' %}</th>
-		            <th class="govuk-table__header" scope="col">{% lcs 'third_parties.ThirdPartiesPage.Variables.DOCUMENT' %}</th>
+					<th class="govuk-table__header" scope="col">{% lcs 'applications.ThirdPartiesPage.Variables.NAME' %}</th>
+					<th class="govuk-table__header" scope="col">{% lcs 'applications.ThirdPartiesPage.Variables.TYPE' %}</th>
+                    <th class="govuk-table__header" scope="col">{% lcs 'applications.ThirdPartiesPage.Variables.ROLE' %}</th>
+					<th class="govuk-table__header" scope="col">{% lcs 'applications.ThirdPartiesPage.Variables.WEBSITE' %}</th>
+					<th class="govuk-table__header" scope="col">{% lcs 'applications.ThirdPartiesPage.Variables.ADDRESS' %}</th>
+					<th class="govuk-table__header" scope="col">{% lcs 'applications.ThirdPartiesPage.Variables.COUNTRY' %}</th>
+		            <th class="govuk-table__header" scope="col">{% lcs 'applications.ThirdPartiesPage.Variables.DOCUMENT' %}</th>
```
Comment (efahy) [inline templates/applications/parties/third-parties.html]: Looks like this is indented with a mixture of tabs and spaces.  On a recent PR for Mark S we decided the team is standardising on tabs - can you indent everything in here with tabs.  Check with Mark about the relevant Pycharm option

File: ui_automation_tests/step_defs/test_submit_standard_application.py [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+from shared.tools.wait import wait_for_download_button, wait_for_element
+from pages.application_overview_page import ApplicationOverviewPage
+from pages.shared import Shared
+from pages.ultimate_end_users_list_page import ThirdPartyListPage
+
+scenarios("../features/submit_standard_application.feature", strict_gherkin=False)
+
+
+@then("good is added to application")
+def good_is_added(driver, context):
```
Comment (efahy) [inline ui_automation_tests/step_defs/test_submit_standard_application.py]: Why is github saying this is all green Benny?

File: applications/views/parties/base.py (hunk 1 above)
Comment (BenAndrew-Kainos) [reply applications/views/parties/base.py]: Created an extended view instead

File: templates/applications/parties/third-parties.html (hunk 2 above)
Comment (BenAndrew-Kainos) [reply templates/applications/parties/third-parties.html]: Done

File: ui_automation_tests/step_defs/test_submit_standard_application.py (hunk 3 above)
Comment (BenAndrew-Kainos) [reply ui_automation_tests/step_defs/test_submit_standard_application.py]: Move my steps into the new step_def and deleted this one

**Pull request merged**